minos/scripts/calculate_cdslen.py

```diff
@@ -14,8 +14,20 @@ def read_fasta(f):
 			yield sid, "".join(seq)
 
 	with open(_out, "w") as fout:
-		for sid, seq in read_fasta(_in):
+		cdna_set = set()
+		for sid, seq in read_fasta(_in_cdna):
+			if sid not in cdna_set:
+				cdna_set.add(sid)
```

Comment:
the `if` is not necessary

can just do : `cdna_set = set(sid for sid, seq in read_fasta(_in_cdna))`

Reply:
I agree